config/runtime.exs

```diff
@@ -12,13 +12,15 @@ external_url = System.get_env("EXTERNAL_URL", "http://localhost:4000")
 
 # Enable Forwarded headers, e.g 'X-FORWARDED-HOST'
 proxy_forwarded = FzString.to_boolean(System.get_env("PROXY_FORWARDED") || "false")
+trusted_proxy = FzString.to_cidr_list(System.get_env("TRUSTED_PROXY") || "[]")
```

Comment:
I think this would just be an IP list? Or did you encounter any situations where a CIDR range was needed to specify the source of the downstream proxy?

Reply:
Since it's [supported by the library](https://hexdocs.pm/remote_ip/RemoteIp.Options.html#module-proxies) I figured it might be useful to someone. Maybe in some case where you load-balance between multiple downstream proxies reaching firezone?